### PR TITLE
Feature/marketprotocol@1.2.0

### LINKED ISSUE #208: update market-solidity to v1.2.0
## Before you `start work`

Please read our contribution [guidelines](https://docs.marketprotocol.io/#contributing) and if there is a bounty involved  please also see [here](https://docs.marketprotocol.io/#gitcoin-and-bounties)

If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please make sure to comment in the issue here immediately after starting work so we know your plans for implementation and a timeline.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.

## Why Is this Needed?
*Summary*: To allow for us to use the new token functionality available in v1.2.0

### Description

*Type*: Feature

### Current Behavior
v1.0.1

### Expected Behavior
v1.2.0

### Solution
[comment]: # (Provide a summary of the solution and a task list on what needs to be fixed.)
*Summary*: 
```
npm uninstall --save market-solidity
npm install @marketprotocol/marketprotocol --save
```

### Definition of Done

- [ ] update npm pkg
- [ ] ensure all tests are passing

### PR DESCRIPTION
**Pull request by pelsasser**
##### Description
Upgrades marketprotocol, abis, marketjs and addresses to use new collateral tokens that allow for faucet like functionality. 

##### Checklist
- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Testing
Loaded up, explorer works, deployed a new contract, works.

##### Refers/Fixes
Fixes #208 - thanks @perfectmak for half of this PR!

### CODE REVIEW

**Comment by pelsasser** [comment]
whoops, hold on, adding one more piece

**Comment by pelsasser** [comment]
okay, all done.

File: package.json [hunk 1]
```diff
@@ -31,7 +31,9 @@
     "web3-fake-provider": "^0.1.0"
   },
   "dependencies": {
+    "@marketprotocol/abis": "0.0.1",
     "@marketprotocol/marketjs": "0.0.3",
```
Comment (rojosnow) [inline package.json]: Did you want to peg this to marketjs `0.0.3`?

**Comment by rojosnow** [comment]
Overall, looks ok with the exception of the one comment.

File: package.json (hunk 1 above)
Comment (pelsasser) [reply package.json]: odd I swore I updated that, thanks and good catch revising now.

**Comment by pelsasser** [comment]
@eswarasai - can you help me with these tests so we can get this merged?

**Pull request merged**